### PR TITLE
Adds Kevlar Armor Plating (AKA The Great Sec Buff)

### PR DESCRIPTION
**Pull request by ynot01**
# GO TO WIKI SECTION FOR NUMBERS

# Vest & Helmet

Adds plating vest & plating helmet to the secdrobe, 6 roundstart should be enough for any sec who wants it

The plating slotted vest & helmet are very weak (5 melee armor only) without plating

The vest is a small item due to essentially being some loose fabric

Vest & helmet become bulky when loaded with a plate

# Philosophy

**These are designed to be direct upgrades to the roundstart armors** in order to give security some progression & reward smart play

I have stared at these numbers for a long time and I believe they are reasonable, especially when you look at other armor crew has access to while security would normally just cope with their roundstart armor

You can do different vest/helmet combos if you think someone is going to go ham on specifically your chest or head. Won't be effective against someone paying attention but it could be a fun risk! Slowdown is applied on helmet and vest individually, so you can go all out on your vest armor and pray that no one aims for your head and you won't move as slowly as you would with both armor pieces.

You can slot in a plate by clicking on a vest/helmet with the plate in your active hand

You can remove a plate by using the vest/helmet in hand

# Plating Design

Platings can be freely printed at security protolathe once researched, but they are fairly expensive (see wiki section)

### MK.I bluespace plating
- Go a bit faster than normal
- Less armor than default sec armor
- Unlocked by weaponry node
- Requires plastic & bluespace crystals

### MK.II ceramic plating
- No movespeed modifier
- A little more armor than default sec armor
- Unlocked by adv. weaponry node
- Comparatively cheap

### MK.III plasteel plating
- Go about riot armor speed
- A good bit more armor
- Unlocked by weaponry node
- Requires miners to not die after first back

### MK.IV titanium plating
- Go slow as shit
- Actual walking tank
- Unlocked by adv. weaponry node
- Very expensive
- Bulky item

##

Weaponry node is now locked behind Basic Sec tech (Science always rushes weaponry because they are full of validhunters and always skip sec tech???? costs 1000 points????? oh right bc the sec isnt supposed to have the strong kill gear, science is)

# Spriting

## Vest
item

worn

## Helmet
item

worn

## Plates

by MakoCyktor

# Wiki Documentation

Weaponry node is now locked behind Basic Sec tech

Plates Mk1 and 3 are unlocked by Weaponry node
Plates Mk2 and 4 are unlocked by Advanced Weaponry node

Plates SET the vest's armor, they do not ADD it
A MK1 plate vest has 20 melee armor, not 25

MK1
- MELEE = 20, BULLET = 20, LASER = 25, ENERGY = 5, BOMB = 5, BIO = 0, RAD = 0, FIRE = 40, ACID = 40, WOUND = 10
- /datum/material/iron = 10000, /datum/material/silver = 5000, /datum/material/plastic = 5000, /datum/material/bluespace = 1500
- Weaponry node
- Slowdown is -0.075 per piece (-0.15 together)

MK2
- MELEE = 35, BULLET = 35, LASER = 35, ENERGY = 15, BOMB = 25, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 20
- /datum/material/iron = 15000, /datum/material/silver = 5000, /datum/material/plasma = 7500
- Adv. Weaponry node
- Slowdown is 0 per piece (0 together)

MK3
- MELEE = 45, BULLET = 45, LASER = 45, ENERGY = 25, BOMB = 30, BIO = 0, RAD = 0, FIRE = 60, ACID = 60, WOUND = 35
- /datum/material/iron = 20000, /datum/material/titanium = 7500, /datum/material/gold = 5000, /datum/material/uranium = 2500, /datum/material/plasma = 7500
- Weaponry node
- Slowdown is 0.15 per piece (0.3 together)

MK4
- MELEE = 60, BULLET = 60, LASER = 60, ENERGY = 40, BOMB = 40, BIO = 0, RAD = 0, FIRE = 75, ACID = 75, WOUND = 50
- /datum/material/iron = 40000, /datum/material/titanium = 30000, /datum/material/gold = 15000, /datum/material/diamond = 15000
- Adv. Weaponry node
- Slowdown is 0.4 per piece (0.8 together)
- Bulky item

## COMPARISONS (NOT CHANGES)
Default Sec Armor (helmet has 5 more melee)
- 0 slowdown
- MELEE = 30, BULLET = 30, LASER = 30, ENERGY = 10, BOMB = 25, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 15

Bulletproof Armor
- 0 slowdown
- MELEE = 15, BULLET = 60, LASER = 10, ENERGY = 10, BOMB = 40, BIO = 0, RAD = 0, FIRE = 50, ACID = 50, WOUND = 20

HoS Hardsuit
- 1 slowdown (as with all space suits)
- MELEE = 45, BULLET = 30, LASER = 30, ENERGY = 10, BOMB = 25, BIO = 100, RAD = 50, FIRE = 95, ACID = 95, WOUND = 25
- Spaceproof

Security Hardsuit
- 1 slowdown (as with all space suits)
- MELEE = 35, BULLET = 25, LASER = 30, ENERGY = 10, BOMB = 10, BIO = 100, RAD = 50, FIRE = 75, ACID = 75, WOUND = 20
- Spaceproof

Riot Armor
- 0.33 slowdown
- MELEE = 50, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, RAD = 0, FIRE = 80, ACID = 80, WOUND = 30

Flagellant's Robes
- -0.6 slowdown
- MELEE = -45, BULLET = -45, LASER = -45,ENERGY = -45, BOMB = -45, BIO = -45, RAD = -45, FIRE = 0, ACID = 0

# Changelog

:cl:  ynot01, MakoCyktor
rscadd: Added kevlar plating which can be printed at a seclathe once researched and loaded into vests bought at SecDrobe
tweak: Locked weaponry behind basic sec tech
/:cl:

### CODE REVIEW

**Comment by Hislyna** [comment]
Henry hasn't even been in power for a day and we already have sec buffs

**Comment by ghost** [comment]
Time to get annihilated by a mech-man with a deathsquad level armor and a stunbatton in each hand

**Comment by KillerOrcaCora** [comment]
It looks like play dough >;)

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -397,3 +397,112 @@
...
+// armor = list(MELEE = 50, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, RAD = 0, FIRE = 80, ACID = 80, WOUND = 30)
+//////////////////////////////////////////////////////////////////////////////
+/obj/item/kevlar_plating
+	name = "debug plating"
+	desc = "You shouldn't see this!"
+	var/name_set = "debug"
+	var/desc_add = " It has incredibly robust pieces of adminium slotted into it."
+	var/slowdown_set = 0 // Slowdown value to set on the vest, for reference a hardsuit has "1" slowdown
+	var/armor_set = list( // Armor value to set on the vest
+		MELEE = 100, BULLET = 100, LASER = 100, ENERGY = 100, BOMB = 100, BIO = 100, RAD = 100, FIRE = 100, ACID = 100, WOUND = 100)
```
Comment (tattax) [inline code/modules/clothing/suits/armor.dm:476]: what if force based on level

**Comment by DeadlyBag** [comment]
Very cool PR, not quite sure how i feel about the MK4 suit having 60 Melee, bullet & laser armor seems very high taking into account potential nanite & dna vault armor. But im hyped to test these out on the server hope it gets through

File: code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (ynot01) [reply code/modules/clothing/suits/armor.dm:476]: what?

Comment (tattax) [reply code/modules/clothing/suits/armor.dm:476]: item force

Comment (ynot01) [reply code/modules/clothing/suits/armor.dm:476]: they aren't weapons

Comment (tattax) [reply code/modules/clothing/suits/armor.dm:476]: yeah but

**Comment by ilikesaying** [comment]
since it's a vest, does it leave the arms and legs exposed?

**Comment by ynot01** [comment]
> since it's a vest, does it leave the arms and legs exposed?

yes

**Review by KillerOrcaCora — APPROVED** [review]
The Emir has spoken

**Comment by Addust** [comment]
titanium plating should come with a shittier sec RPG that fires weaker rockets that don't like breaching walls as much and don't cause hull breaches

because walking tank
should be a tank
not a shitty disabler march

**Comment by JamieD1** [comment]
Does it really need to be called bluespace armour
Surely Kevlar or something else is better

**Comment by ynot01** [comment]
> Does it really need to be called bluespace armour Surely Kevlar or something else is better

initially they were all called kevlar but seeing as none of the items are actually made of kevlar i renamed them all, to be better suited to what they are made of

mk1 armor requires bluespace and has a negative movespeed modifier so to distinguish it i named it "bluespace"

**Comment by ynot01** [comment]
> titanium plating should come with a shittier sec RPG that fires weaker rockets that don't like breaching walls as much and don't cause hull breaches
> 
> because walking tank should be a tank not a shitty disabler march

out of the scope of this pr

**Comment by ynot01** [comment]
help

**Pull request merged**